src/libpriv/rpmostree-postprocess.c

```diff
@@ -1442,6 +1450,12 @@ rpmostree_treefile_postprocessing (int            rootfs_fd,
       if (!glnx_shutil_rm_rf_at (rootfs_fd, val, cancellable, error))
         return FALSE;
     }
+  if (len > 0)
```

Comment:
But the array could be specified but empty, right? I'd just shove this whole loop with the pre and post `/etc` moves in the if statement itself.